src/App.jsx

```diff
@@ -9,36 +10,19 @@ export const tabs = [
   { id: 'tab-3', title: 'Tab 3', content: 'Some text 3' },
 ];
 
-export const App = () => (
-  <div className="section">
-    <h1 className="title">Selected tab is Tab 1</h1>
+export const App = () => {
+  const [activeTabId, onTabSelected] = useState(tabs[0].id);
```

Comment:
Use the standard function name to change the state of `setActiveTabId` to avoid confusion.
```suggestion
  const [activeTabId, setActiveTabId] = useState(tabs[0].id);
```